style.css

```diff
@@ -1,62 +1,215 @@
-body {
-  background: url('aust copy.jpg');
-  background-size: cover;
-  /* position: fixed; */
+#city {
+	background: url('building.png');
+	background-size: cover;
+	min-height: 100vh;
+	position: relative;
+}
+.nameDev {
+	/* background: rgba(204, 204, 204, 0.5); opacity goes all the way from edge to edge */
+}
+.toplinework {
+	/* this div holds all of the top line work*/
+	margin-bottom: 0px;
+	height: 50px;
 }
 
-.splash {
-  font-family: 'Noto Sans', sans-serif;
-  padding-top: 60px;
+.tL1 {
```

Comment:
Cool usage of CSS to style your name. You don't need to use an image, don't need to load another asset, it's efficient!

These styles are very repetitive, though. The whole `.tL<number>` series have these traits in common:
```css
border-top: 2px solid white;
margin-top: 5px;
margin-bottom: 5px;
```
The only variation is their `width: <percentage>` property.

You can make a class that will encompass the *common* CSS properties for those lines, and then have some class for the varying widths, then you can combine them. You'll save yourself a lot of lines of CSS.

You can apply these to the `.bL<number>` as well.

Reply:
I streamlined the code for the classes.  I consolidated all the common traits to one .line class.  The widths are being controlled via ID in CSS.  I deleted all of the bottom line code and used the same common lines within a unique div .bottomlinework which I rotated.
<img width="514" alt="Screen Shot 2021-01-24 at 3 38 42 PM" src="https://user-images.githubusercontent.com/51690062/105644241-3ee20700-5e5a-11eb-9046-03410f508f9d.png">

Reply:
Nice. 

My only suggestion is this:

You have more than one element calling an `id`, and there should only be one `id="l6"` per page.

Essentially, what you've done, is create *utility* classes for width. What you can do then, is convert those `#l<number>` to something like `.w-10` (for a `width: 10%`) that you can reuse at-will.

Bootstrap has something similar with their `.w-25` class.